### PR TITLE
Check Service owner for pgAdmin ServiceName

### PR DESCRIPTION
**Pull request by jmckulk**
**Checklist:**

 
 - [ ] Have you added an explanation of what your changes do and why you'd like them to be included?
 - [ ] Have you updated or added documentation for the change, as applicable?
 - [ ] Have you tested your changes on all related environments with successful results, as applicable?
   - [ ] Have you added automated tests?

**Type of Changes:**

 
 - [ ] New feature
 - [ ] Bug fix
 - [ ] Documentation
 - [ ] Testing enhancement
 - [ ] Other

**What is the current behavior (link to any open issues here)?**

**What is the new behavior (if this is a feature change)?**
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

**Other Information**:

### CODE REVIEW

File: testing/kuttl/e2e/standalone-pgadmin-service/10--manual-service.yaml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# Manually create a service that should be taken over by pgAdmin
+# The manual service is of type LoadBalancer
+# Cnce taken over, the type should change to ClusterIP
```
Comment (jmckulk) [inline testing/kuttl/e2e/standalone-pgadmin-service/10--manual-service.yaml:3]:
```suggestion
# Once taken over, the type should change to ClusterIP
```

File: testing/kuttl/e2e/standalone-pgadmin-service/21-steal-service.yaml
Comment (jmckulk) [inline testing/kuttl/e2e/standalone-pgadmin-service/21-steal-service.yaml:1]: probably will rename this file - at the very least to match the `21--name` convention

File: testing/kuttl/e2e/standalone-pgadmin-service/21-assert.yaml [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+spec:
+  selector:
+    postgres-operator.crunchydata.com/pgadmin: pgadmin-service-owner
+  ports:
+  - port: 5050
+    targetPort: 5050
+    protocol: TCP
+    name: pgadmin-port
+  type: ClusterIP
+---
```
Comment (jmckulk) [inline testing/kuttl/e2e/standalone-pgadmin-service/21-assert.yaml:23]:
```suggestion
---
# An event should be created for the failure to reconcile the Service
```

**Review by dsessler7 — APPROVED** [review]
LGTM

**Pull request merged**